serialized_data_converter.py

```diff
@@ -349,6 +349,7 @@ def convert(self, edit):
                 self.output = yaml.yaml_dumps(
                     self.plist,
                     default_flow_style=default_flow_style,
+                    indent=self.settings.get("yaml_indent"),
```

Comment:
I know, but I plan on removing the magic numbers with default defines in the future and consolidating them in one place.  It will make it easier on me if they are all on the same level.